Gems/HttpRequestor/Code/Include/HttpRequestor/HttpRequestParameters.h

```diff
@@ -36,30 +50,49 @@ namespace HttpRequestor
         Parameters(Parameters&&) = default;
         Parameters& operator=(Parameters&&) = default;
 
-        //returns the URI in string form as an recipient of the HTTP connection
-        const Aws::String& GetURI() const { return m_URI; }
-
-        //returns the method of which the HTTP request will take. GET, POST, DELETE, PUT, or HEAD
-        Aws::Http::HttpMethod GetMethod() const { return m_method; }
-
-        //returns the list of extra headers to include in the request
-        const Headers & GetHeaders() const { return m_headers; }
-
-        //returns the stream for the body of the request
-        const std::shared_ptr<std::stringstream> & GetBodyStream() const { return m_bodyStream; }
-
-        //returns the function of which to feed back the JSON that the HTTP call resulted in. The function also requires the HTTPResponseCode indicating if the call was successful or failed
-        const Callback & GetCallback() const { return m_callback; }
+        //! Get the URI in string form as an recipient of the HTTP connection.
+        const Aws::String& GetURI() const
+        {
+            return m_URI;
+        }
+
+        //! Get the HTTP method configured to use for a request.
+        Aws::Http::HttpMethod GetMethod() const
```

Comment:
Nit: Same as above. Do we want to add comment for the return value?

Reply:
@sptramer asked me to remove the return value documentation as it added nothing for simple types.